src/components/dialog/dialog/dialog.ts

```diff
@@ -363,101 +402,61 @@ export class SbbDialogElement extends SbbNegativeMixin(LitElement) {
 
   // Set focus on the first focusable element.
   private _setDialogFocus(): void {
-    const firstFocusable = this.shadowRoot!.querySelector(IS_FOCUSABLE_QUERY) as HTMLElement;
+    const firstFocusable = getFirstFocusableElement(
+      Array.from(this.children).filter((e): e is HTMLElement => e instanceof window.HTMLElement),
```

Comment:
Don't we exclude close button with that?

Reply:
In this case not anymore, as we are now slotting the close button with the `sbb-dialog-title`.

Reply:
Ah I see because the focusable function goes into the shadow DOM of sub components.